.gitignore

```diff
@@ -25,3 +25,5 @@
 /var/
 /vendor/
 ###< symfony/framework-bundle ###
+/tools/phpcs/composer.json
```

Comment:
files can be removed, we dropped phpcs and rector

Reply:
Done